packages/swap/src/server.ts

```diff
@@ -31,7 +31,7 @@ const appRouter = router({
     .mutation((v) => openSwapDepositChannel(v.input)),
   encodeVaultSwapData: publicProcedure
     .input(encodeVaultSwapDataSchema)
-    .query((v) => encodeVaultSwapData(v.input)),
+    .mutation((v) => encodeVaultSwapData(v.input)),
```

Comment:
while this is not really a mutation, registering it as one makes it consistent to the `openSwapDepositChannel` method and allows us to use the same configuration for stuff like rate limiting